submissions/NastjonkaK/OOP-task/js/app.js

```diff
@@ -0,0 +1,128 @@
+// Enemies our player must avoid
+let points = 0;
+let scoreboard = document.createElement('p');
+scoreboard.textContent = 'Your score: 0';
```

Comment:
I think you can make function which will apply points and render score in scoreboard.

```
const renderScore = points => { scoreboard.innerHTML = `Your score: ${points}` };
```